docs/makefile.md

```diff
@@ -0,0 +1,79 @@
+# Makefile
+
+Here is a Makefile template. It provides some shortcuts for the most common tasks.
+To use it, create a new `Makefile` file at the root of your project. Copy/paste
+the content in the template section. To view all the available commands, run `make`.
+
+For example, in the [getting started section](/README.md#getting-started), the
+`docker-composer` commands could be replaced by:
+
+1. Run `make build` to build fresh images
+2. Run `make up` (detached mode without logs)
+3. Run `make down` to stop the Docker containers
+
+Of course, this template is basic for now. But, as your application is growing,
+you will probably want to add some targets like running your tests as described
+in [the Symfony book](https://symfony.com/doc/current/the-fast-track/en/17-tests.html#automating-your-workflow-with-a-makefile).
+You can also find a more complete example in this [snippet](https://www.strangebuzz.com/en/snippets/the-perfect-makefile-for-symfony).
+
+**PS**: If using Windows, you have to install [chocolatey.org](https://chocolatey.org/)
+or use [Cygwin](http://cygwin.com) to use the `make` command. Check out this
+[StackOverflow question](https://stackoverflow.com/q/2532234/633864) for more explanations. 
+
+## The template
+
+```Makefile
+# Executables (local)
+DOCKER_COMP = docker-compose
+
+# Docker containers
+PHP_CONT = $(DOCKER_COMP) exec php
+
+# Executables
+PHP      = $(PHP_CONT) php
+COMPOSER = $(PHP_CONT) composer
+SYMFONY  = $(PHP_CONT) bin/console
```

Comment:
Hello,
IMO it seems more logical to write :
```suggestion
SYMFONY  = $(PHP) bin/console
```
Moreover, in my case, I had the following error trying to use it as is :
`OCI runtime exec failed: exec failed: unable to start container process: exec: "bin/console": permission denied: unknown
make: *** [Makefile:50: sf] Error 126`
Thanks for reading this,
Bye.

Reply:
LGTM. Would you mind opening a PR?

Reply:
Sorry, I don't know GitHub very well, thought you could just accept the suggestion from here...
I've opened a PR as asked !

Reply:
LGTM too, there is rather a problem elsewhere. The Symfony command must be executed inside the php container and not on the machine that does not have the same PHP context.